src/test/java/seedu/address/logic/commands/CommandResultTest.java

```diff
@@ -60,20 +60,60 @@ public void isShowHelp_showHelpTrue_true() {
     }
 
     @Test
-    public void isShowHelp_listMeetingCommandExecute_false() {
+    public void isShowHelp_showHelpFalse_false() {
         CommandResult cr = new CommandResult("", false, false);
         assertFalse(cr.isShowHelp());
     }
 
     @Test
-    public void isExit_exitCommandExecute_true() {
+    public void isExit_exitTrue_true() {
         CommandResult cr = new CommandResult("", false, true);
         assertTrue(cr.isExit());
     }
 
     @Test
-    public void isExit_exitCommandExecute_false() {
+    public void isExit_exitFalse_false() {
         CommandResult cr = new CommandResult("", false, false);
         assertFalse(cr.isExit());
     }
+
+    @Test
+    public void isClientSpecific() {
```

Comment:
I'm not too sure about the convention here. From the website they suggest we use the naming convention similar to `nameOfFunction_Input_Result`. So I was thinking should we split this up into 2 different methods, `isClientSpecific_validClientSpecific_success` and `isClientSpecific_invalidClientSpecific_faillure`. What do you think?

Reply:
hmm i found that this method was too simple to split up into 2 methods. i noticed that `equals` being tested also did not split up, so i decided not to split into different methods for these as they will end up being unnecessarily long and tedious to write and for not much more purpose.

Reply:
would like to hear your opinions on this thanks!

Reply:
Ah ok, yea then i think it should be ok.